.github/workflows/deploy-ghpages-latest-stable.yml

```
@@ -112,7 +112,7 @@ jobs:
         URL="https://qibogang.github.io/qibo/${missing_doc}/${missing_doc}"
         mkdir "$missing_doc"
         # Check if it is possible to connect to the website 
-        if wget "${URL}" 2>/dev/null; then 
+        if wget "${URL}"; then 
```

Ah no, wait, I missed that the redirect condition was used in the condition.
Sorry, my bad.

In this case, you're downloading twice, that's why there was the `--spider` option (it only traverses the links, but without downloading anything) and the redirect (because you're just testing, you do not want to actually download anything).

So, my advice would be to actually attempt the download, with just the simple `wget` call, and then test for file existence, so something like
```suggestion
        wget "${URL}"
        if [ -f $(echo $URL  | rev | cut -d'/' -f1 | rev) ]; then 
```

---

The command is just an example, test it on your machine with an actual URL